mlx-mkbfb

```diff
@@ -594,6 +596,65 @@ def dump_stream(infn, do_dump, do_extract, prefix):
     inf.close()
 
 
+def strip_images(infn, outfn, version):
+    imgs = []
+
+    with open(infn, 'rb') as inf:
+        while True:
+            try:
+                inimg = Image(instream=inf)
+                imgs.append(inimg)
+            except NoHdr:
+                break
+
+    # Filter out version we don't want
+    imgs = [img for img in imgs if
+            img.get_image_ver() != version]
```

Comment:
I don't think this is quite right.  Not every image will have a separate copy for each chip.  When loading images, we select the one with the largest version <= chip_id.   It assumes that images with multiple versions appear in the bfb in ascending order of image_ver.

If I understand this code right, if you do a "-s 0" it will strip out more than we want since most things which are common will have image_ver 0.

Reply:
Yeah, that wouldn't be good. I've reworked this option into a more general --filter option, which is careful about this sort of thing by default.